source/guides/stacks/hpcrs.rst

```diff
@@ -0,0 +1,225 @@
+
+      go get -d github.com/sylabs/singularity
+
+
+   .. note::
+
+      Go will complain that there are no Go files, but it will still download the Singularity source code to the appropriate directory within the $GOPATH.
+
+#. Checkout version 3.0.0 of Singularity
+
+   .. code-block:: bash
+
+      export VERSION=v3.0.3 # or another tag or branch if you like && \
+      cd $GOPATH/src/github.com/sylabs/singularity && \
+      git fetch && \
+      git checkout $VERSION # omit this command to install the latest bleeding edge code from master
+
+
+#. Build Singularity
+
+   Singularity uses a custom build system called makeit. mconfig is called to generate a Makefile and then make is used to compile and install.
+   The devpkg-openssl, devpkg-util-linux  package may be required and can be installed using the :command:`sudo swupd bundle-add <pkg-name>`
+
+   .. code-block:: bash
+
+      ./mconfig && \
+      make -C ./builddir && \
+      sudo make -C ./builddir install
+
+
+#. Configure bash completion (optional)
+
+   To enjoy bash completion with Singularity commands and options, source the bash completion file. Add this command to your ~/.bashrc file so that bash completion continues to work in new shells
+
+   .. code-block:: bash
+
+      . /usr/local/etc/bash_completion.d/singularity
+
+
+Converting Docker images to Singularity Images
+**********************************************
+
+#. Download d2s
+
+   :command:`d2s` os an open source tool to convert Docker images to Singularity images. You can use the script in the location where it is downloaded, or install it using the included :file:`setup.py` file with the :command:`python setup.py install`
+
+
+   .. code-block:: bash
+
+      git clone https://github.com/intel/stacks.git
+      cd hpcrs/d2s
+
+#. List local Docker images
+
+   .. code-block:: bash
+
+      python d2s.py --list_docker_images
+
+   Your output can appear like this:
+
+   .. code-block:: console
+
+      ==============================
+      Docker images present locally
+      ==============================
+      ID         NAME
+      0: clearlinux/stacks-dlrs-mkl
+      1: clearlinux/stacks-dlrs_2-mkl
+      ==============================
+
+#. Convert to Singularity images
+
+   To convert the Docker images to Singularity images, use the :command:`d2s` script with the ID numbers of the images you wish to convert.
+
+
+   .. code-block:: bash
+
+      python d2s.py --convert_docker_images <ID_1> <ID_2>
+
+#. Use the Singularity image
+
+   To use the container shell to run workloads, launch the image and you will be dropped into the shell. The Singularity image name will be the same as the name of the Docker image, with slashes converted to underscores.
+
+   .. code-block:: bash
+
+      singularity shell <singularity image>
+
+    Using the example output above, after conversion you could launch the clearlinux/stacks-dlrs-mkl Singularity image with :command:`singularity shell clearlinux_stacks-dlrs-mkl`
```

Comment:
Could you split Ln 143 in two? 

It's easier to confirm if I don't have to scroll far right to see command:
 `singularity shell clearlinux_stacks-dlrs-mkl`